woocommerce/woocommerce-android
# [Blaze] Fix date formatting for the campaign creation API request

Closes: #12370


### Description
This PR fixes the issue of the parent issue: which is to ensure the date formatting for the Blaze campaign request respects the ISO standard by using the regular Arabic numerals always.

This PR just updates FluxC, the actual fix is in FluxC PR https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/3085

### Steps to reproduce
1. Using `trunk`
2. Switch your device (or the app) to the Arabic language (Choose **Arabic(Egypt) العربية(مصر)** to ensure using Indo-Arabic numerals).
3. Attempt to create a Blaze campaign.
4. Notice the failure.

### Testing information
1. Switch to this branch
2. Switch your device (or the app) to the Arabic language (Choose **Arabic(Egypt) العربية(مصر)** to ensure using Indo-Arabic numerals).
3. Attempt to create a Blaze campaign.
4. Confirm the creation succeeded.

### Images/gif



## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:
